openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -156,6 +156,204 @@ data:
         - name: v1alpha1
           served: true
           storage: true
+    - apiVersion: apiextensions.k8s.io/v1beta1
+      kind: CustomResourceDefinition
+      metadata:
+        name: knativeservings.serving.knative.dev
```

Comment:
Not only dup of below CRD, but also this CRD is same name of https://github.com/openshift/knative-serving/pull/202/files#diff-99f5ab4373a5da5b5ee5414aa78fc248R75. So, it causes an error.